test/libyul/yulOptimizerTests/unusedFunctionReturnParameterPruner/pop.yul

```diff
@@ -0,0 +1,32 @@
+// A test to see if the optimization step can deal with pop
+// Expression splitter converts `pop(f(a))` into
+// {
+//   let z := f(a)
+//   pop(z)
+// }
+// Unless `pop(z)` is removed, `f` cannot be rewritten.
+{
+    let a := sload(1)
+    let z := f(a)
+    pop(z)
```

Comment:
The step currently removes all `pop(identifier)` at the beginning. So this test works.